esp/templates/themes/editor.html

```diff
@@ -121,6 +121,15 @@ <h3 class="accordion-toggle" data-toggle="collapse" href="#scaffolding" rel="pop
                             </div>
                         </div>
 
+                    {% if theme_name != "bigpicture" and theme_name != "barebones" %}
+                        <div class="control-group">
+                            <label class="control-label" for="contentbg">Content Background:</label>
+                            <div class="controls">
+                                <input type="text" id="contentbg" name="contentBackground" class="color" value="{{contentBackground}}"/>
+                            </div>
+                        </div>
+                    {% endif %}
+
                         <div class="control-group">
```

Comment:
What are the "well" and "hero unit" backgrounds? I couldn't find anywhere that those were used in the themes I checked. Should those also be hidden for some/all themes?

Reply:
"well" is used for all themes on the themes landing page "http://localhost:8000/themes/" at the very least, I'm not sure if it's used elsewhere. 

Hero unit is used on the front page of barebones, but based on a github search, I don't think it's used elsewhere. I'll hide it for other themes-- thanks for catching that!

Reply:
Hmm...do we really need folks to be able to customize the themes landing page appearance?

Reply:
Hmm, I agree that it's probably not necessary. I'll remove that option to clean things up.